src/eureka/S1_detector_processing/ramp_fitting.py

```diff
@@ -87,6 +89,46 @@ def process(self, input):
             Updated for JWST version 1.3.3, code restructure
         '''
         with datamodels.RampModel(input) as input_model:
+
+            if self.s1_meta.update_sat_flags:
```

Comment:
In general, I try to do the following to reduce the annoyance of users when they update their code and suddenly need to set ECF parameters that they don't care about:

```
if hasattr(self.s1_meta, 'update_sat_flags') and self.s1_meta.update_sat_flags:
```

Reply:
updated all of these lines throughout